### ISSUE TITLE
Uses browsertime.har.gz instead of browsertime.har

### ISSUE THREAD

**Issue by SunnyGurnani** [sitespeedio/compare#109]
When inferrring Har Path from PageURL Compare is using browsertime.har.gz whereas SiteSpeed.io is storing with browsertime.har

Test with Following
https://dobermanv2.s3-us-west-2.amazonaws.com/sitespeed-result/2020/10/19/xGIVIfF4e-control/pages/www.realtor.com/index.html

**Comment by soulgalore** [comment]
Hi @SunnyGurnani I didn't fully get where its wrong? Is the compare button where the link is broken? Can you show me with a screenshot?

**Comment by SunnyGurnani** [comment]
![Uploading image.png…]()

**Comment by soulgalore** [comment]
Aha thanks so we take for granted its gzipped. I guess the workaround then is to try to fetch both and if the first fail, try the other one.

**Comment by yashgarde** [comment]
Was this ever addressed because I believe I am still encountering this on compare.sitespeed.io when providing URL's to sitespeed results.

**Comment by soulgalore** [comment]
@yashgarde no, easiest for now is to use  `--gzipHAR` in sitespeed.io so the HAR files are gzipped.
